Resources/Private/JavaScript/@types/typo3/AjaxRequest.d.ts

```
@@ -0,0 +1,15 @@
+declare module "@typo3/core/ajax/ajax-request.js" {
+
+  declare interface AjaxRequest {
+
+    new(url: string): AjaxRequest;
```

Biome (1.9.4)

[error] Don't use the new method in interfaces.

new in an interface suggests it's instantiable, which is incorrect. The returned type should different from the constructor's type.

(lint/suspicious/noMisleadingInstantiator)